style/template.tex

```diff
@@ -41,7 +41,7 @@
 \ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
   \usepackage[T1]{fontenc}
   \usepackage[utf8]{inputenc}
-$if(euro)$
+\mathbf{$if(euro)$
```

Comment:
Is this missing a `}`?

Reply:
You're right. But strangely enough it compiles nonetheless without errors.

```suggestion
\mathbf{$if(euro)$}
```

With closing bracket it compiles without error 43 (or any other) as PDF.